paymentsheet/src/main/java/com/stripe/android/paymentsheet/elements/common/Mandate.kt

```diff
@@ -0,0 +1,17 @@
+package com.stripe.android.paymentsheet.elements.common
+
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+
+@Composable
+internal fun Mandate(text: String) {
+    Text(
+        text,
+        modifier = Modifier
+            .fillMaxWidth(),
+        color = Color.Gray
+    )
+}
```

Comment:
My lint on Android Studio complains about files not ending with new line. I'll add that rule to `.editorconfig`.

Reply:
thanks